packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -98,6 +99,24 @@ export class Vector2 {
         return result;
     }
 
+	/**

```

Comment:
Can we remove those two functions ?

The goal is to only import math.polar as a type to prevent the addition of polar in math.vector when unsused ?

Reply:
ppl could use the Polar functions for this.